org.freedesktop.Platform.GL.asahi.yml

```diff
@@ -9,6 +9,26 @@ sdk-extensions:
 build-options:
   prepend-path: /usr/lib/sdk/llvm15/bin
   prepend-ld-library-path: /usr/lib/sdk/llvm15/lib
+
+
+x-arch-build-options:
+  x86_64: &x86_64-build-options
+    prefix: /usr/lib/x86_64-linux-gnu/GL/asahi
+    prepend-pkg-config-path: /usr/lib/x86_64-linux-gnu/GL/asahi/lib/pkgconfig
+    libdir: /usr/lib/x86_64-linux-gnu/GL/asahi/lib
+# Needs https://github.com/flatpak/flatpak-builder/issues/381 to be resolved before this can actually be used. Until then, only 64-bit builds.
```

Comment:
super-nit: indent the comments at the level of the line they're commenting :sweat_smile: